network-conduit-tls/Data/Conduit/Network/TLS.hs

```diff
@@ -131,7 +131,7 @@ serverHandshake socket creds = do
         { TLS.serverWantClientCert = False
         , TLS.serverSupported = def
             { TLS.supportedCiphers = TLSExtra.ciphersuite_default
-            , TLS.supportedVersions = [TLS.SSL3,TLS.TLS10,TLS.TLS11,TLS.TLS12]
+            , TLS.supportedVersions = [TLS.TLS10,TLS.TLS11,TLS.TLS12]
```

Comment:
My suggestion was not to remove `TLS.SSL3` and leave the rest. It was to completely leave out the `TLS.supportedVersions` override and let the underlying TLS library (`hs-tls`) provide  a sensible default.  Which it does....

Reply:
I did like this because it is explicit, but yes, its probably better if TLS decides on the default.